<?xml version="1.0"?>

<launch>
  
  <arg name="use_alternate_planner" default="true" />

  <node pkg="hector_move_base" type="hector_move_base" name="move_base" output="screen" respawn="true">
    
    <param name="SBPLLatticePlanner/primitive_filename" value="$(find hector_move_base)/mprim/crab_higher_cost_only_back.mprim" />
    
        <param name="HectorSbplStairsPlanner/primitive_filename" value="$(find hector_sbpl_stairs_planner)/mprim/mprim_obelix_v11_with_backwards.mprim" />  
    
    <param name="use_alternate_planner" value="$(arg use_alternate_planner)" />
    <param name="hector_exploration_planner/explore_close_to_path" value="false" />
    
    <rosparam file="$(find hector_move_base)/params/hector_move_base_params.yaml" command="load" />
    <rosparam file="$(find hector_move_base)/params/global_costmap_params_bertl.yaml" command="load" />
    <rosparam file="$(find hector_move_base)/params/hector_global_planner_params.yaml" command="load" />
    
  </node>
</launch>
